src/index.js

```diff
@@ -4,9 +4,16 @@ import App from "./App";
 import "./index.css";
 import state from "./state";
 
+var currentCategory = 'tires';
+function changeCategory(category) {
+  currentCateogry = category;
```

Comment:
Make sure you app runs with no errors before you turn it in.
what is currentCateogry